test/system/030-run.bats

```diff
@@ -814,10 +814,14 @@ EOF
 }
 
 @test "podman run defaultenv" {
-    run_podman run --rm $IMAGE printenv
+    run_podman run -t=true --rm $IMAGE printenv
     assert "$output" =~ "TERM=xterm" "env includes TERM"
     assert "$output" =~ "container=podman" "env includes container=podman"
 
+    run_podman run --rm $IMAGE printenv
+    assert "$output" != "TERM=xterm" "env doesn't include TERM"
```

Comment:
Careful - this does not do what you think it does! Consider:
```
TERM=something-else
```
or
```
PATH=/usr/bin
TERM=xterm
```
Suggestion: `assert "$output" !~ "TERM=" "env doesn't include TERM`

Reply:
Yes and I believe that 
 run_podman run --tty=false --env TERM=foobar --rm $IMAGE printenv

Should show TERM=foobar